INSTALL.md

```diff
@@ -0,0 +1,118 @@
+# Building from sources
```

Comment:
I'd add a first-level heading for the whole file because this file contains various scenarios, and then a table of contents.